Makefile

```diff
@@ -124,6 +121,18 @@ docker-buildx:
 		-f Dockerfile \
 		. ${PUSH_ARG}
 
+# Pull the docker images for acceptance
+docker-pull: docker-build
```

Comment:
Can we use a bit different style for anything related to dependencies, like `docker-pull-deps`?

`docker-whatever` was used to build/push the controller image until now so I thought it would be nice to make a distinction

Reply:
They also build and push the runner image with this PR, that's fine, right?

Reply:
I renamed it back to acceptance/pull.